src/components/dialog/dialog.js

```diff
@@ -233,6 +234,7 @@ class Dialog extends Modal {
   }
 
   focusDialog() {
+     if (!this._dialog) { return; }
```

Comment:
one too many space in front if `if` - our linter should pick this up. Can you try running `gulp lint` on the code base?

Reply:
My bad. I just changed editors and I am not visually picking up on the lint errors.